### PR TITLE
Done homework 4.

### PR DESCRIPTION
**Pull request by ArmineZak**

### CODE REVIEW

File: src/algorithms/SortingAlgorithms.java [hunk 1]
```diff
@@ -3,45 +3,122 @@
...
+            int minIndex = i;
+            for (int j = i; j < array.length; j++) {
+                if (array[j] < array[minIndex]) {
+                    minIndex = j;
+                    swap(array, minIndex, i);
+                }
+            }
+        }
+    }
+
```
Comment (vahan1994vs) [inline src/algorithms/SortingAlgorithms.java:36]: it doesn't work correctly. Please try again.
https://en.wikipedia.org/wiki/Insertion_sort

**Review by vahan1994vs — COMMENTED** [review]
Pay attention հօw the algorithms work.

File: src/algorithms/SortingAlgorithms.java (hunk 1 above)
Comment (ArmineZak) [reply src/algorithms/SortingAlgorithms.java:36]: Correcred.

Comment (ArmineZak) [reply src/algorithms/SortingAlgorithms.java:36]: Corrected.

**Pull request closed without merging**